Fix dynamic script cortex xdr identity information widget (#28558)
demisto/content: Packs/CortexXDR/ReleaseNotes/5_0_5.md

#### Scripts

##### CortexXDRIdentityInformationWidget

> Added a validation to check if there is only one element in the "AWS IAM users" key.

Author: ```suggestion
Added a validation to check whether there is one element in the **AWS IAM users** key.
```
